v1/api/swagger/components/schemas/AvailableImage.yaml

```diff
@@ -23,6 +24,10 @@ properties:
     description: >-
       Second-level grouping of imaging services.  Typically references a
       specific OS or application.
+  Name:
+    type: string
+    example: HPE VMaaS VMware ESXi
```

Comment:
This example conflicts with the Flavor, Version, and Description example, which is for an Ubuntu image, but the Name is an ESXi image. This makes the generated document look incorrect. Either change the Name example to "HPE Ubuntu 18.04", or change the examples to be an "ESXi" image.

Reply:
[3a2c981](https://github.com/HewlettPackard/hpegl-metal-client/pull/90/commits/3a2c98106876073bb14f701c61a09cb3215b4ecc)